modules/byoh-configuring.adoc

```diff
@@ -45,5 +45,6 @@ data:
   instance.example.com: |-
     username=core
 ----
-<1> An address to SSH in to the instance with. This can be a DNS name or an IPv4 address. A PTR record must exist corresponding to the address used in the config map resolving to the instance host name for successful reverse DNS lookups.
+<1> The address to SSH in to the instance with, either a DNS name or an IPv4 address. A DNS PTR record must exist for this address. It is recommended that you use a DNS name with your BYOH instance if your organization uses DHCP to assign IP addresses. If not, you need to update the `windows-instances` ConfigMap whenever the instance is assigned a new IP address.
```

Comment:
Supplementary Style Guide says not to use SSH as a verb: "Do not use ssh as a verb; for example, write "Use SSH to connect to the remote server" instead of "ssh to the remote server"." 

Could this be reworded as "An address to connect to the instance using SSH, ... "?

Reply:
@bscott-rh How dos this sound:

> 
<1> The address that the WMCO uses to reach the instance over SSH...

Reply:
That sounds good to me Michael.